dq-game/src/client/event_coordinator.py

```diff
@@ -48,18 +53,29 @@ def show_instructions():
         page.set_data(self.renderer.username)
         page.set_callback(show_instructions)
         page.render()
+        if self.front_screen:
+            self.front_screen.show_instructions_and_ready_up()
 
     def on_choose_theme(self):
         themes = self.controller.get_theme_list()
         self.renderer.select_themes(themes, self.controller.send_client_theme_choices)
+        if self.front_screen:
+            self.front_screen.choose_theme()
 
     def on_start_round(self):
         self.current_round_number = self.controller.get_round_number()
         self.renderer.show_round_instructions(self.current_round_number)
+        if self.front_screen:
+            self.front_screen.start_round()
```

Comment:
every event coordinator will call this method and then every front screen will call 

```
    def start_round(self):
        self.socket.send_to_all("START_ROUND", "event")
```

is this correct ? the send to ALL will be bigger than the necessary

Reply:
I use send to all because the front screen has no client name associated with it.